CHANGELOG.md

```diff
@@ -14,19 +14,23 @@ You can find its changes [documented below](#060---2020-06-01).
 - Re-export `druid_shell::Scalable` under `druid` namespace. ([#1075] by [@ForLoveOfCats])
 - `TextBox` now supports ctrl and shift hotkeys. ([#1076] by [@vkahl])
 - Added selection text color to textbox. ([#1093] by [@sysint64])
+- `scroll_component` for ease of adding consistant, customized, scrolling behavior to a widget. ([#1107] by [@ForLoveOfCats])
 
 ### Changed
 
 - `Scale::from_scale` to `Scale::new`, and `Scale` methods `scale_x` / `scale_y` to `x` / `y`. ([#1042] by [@xStrom])
 - Major rework of keyboard event handling. ([#1049] by [@raphlinus])
 - `Container::rounded` takes `KeyOrValue<f64>` instead of `f64`. ([#1054] by [@binomial0])
+- `Scroll` renamed to `AbsoluteScroll`. ([#1107] by [@ForLoveOfCats])
+- `List` handles its own scrolling on either the vertical or horizontal axis. ([#1107] by [@ForLoveOfCats])
 
 ### Deprecated
 
 ### Removed
 
 - `Scale::from_dpi`, `Scale::dpi_x`, and `Scale::dpi_y`. ([#1042] by [@xStrom])
 - `Scale::to_px` and `Scale::to_dp`. ([#1075] by [@ForLoveOfCats])
+- Specific axis scrolling behavior from `AbsoluteScroll`. ([#1107] by [@ForLoveOfCats])
```

Comment:
I removed it just because it made sense at the time with the other changes but I can definitely see the usage cases. Technically a `List` with a single child would work identically but that feels silly.

Reply:
Re-added